packages/flutter/lib/src/material_animated_icons/data/menu_arrow.g.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file is a temporary placeholder for generated animated icon data files,
+// just to demonstrate the proposed package structure.
+part of material_animated_icons;
+
+class _MenuArrowIconData implements _AnimatedIconData {
+  const _MenuArrowIconData();
+
+  @override
+  String get placeHolderIconData => 'menu->arrow';
```

Comment:
this should be a private member, if you want to prevent other classes from implementing it